<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hobby</class>
 <widget class="QMainWindow" name="Hobby">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Привычки</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/icons/img/icon_habbits.png</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ноябрь 2020</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Таблица</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="styleSheet">
           <string notr="true">QTableWidget{
alternate-background-color: rgb(245, 245, 245);
}</string>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="showGrid">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>50</number>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>50</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>40</number>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>40</number>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <row>
           <property name="text">
            <string/>
           </property>
          </row>
          <column>
           <property name="text">
            <string>1</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>2</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>3</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>4</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>5</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>6</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>7</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>8</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>9</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>10</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>11</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>12</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>13</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>14</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>15</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>16</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>17</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>18</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>19</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>20</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>21</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>22</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>23</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>24</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>25</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>26</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>27</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>28</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="habbitEdit">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>Название привычки</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="createButton">
            <property name="text">
             <string>Добавить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteButton">
            <property name="text">
             <string>Удалить</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Графики</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QCustomPlot" name="customPlot" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="themeBox">
            <property name="text">
             <string>Тёмный фон</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QComboBox" name="habbitComboBox">
              <item>
               <property name="text">
                <string>Все привычки</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Бег</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Чтение</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Курсовая</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="graphStyleCombo">
              <item>
               <property name="text">
                <string>Линейная диаграмма 1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Линейная диаграмма 2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Линейная диаграмма 3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Столбчатая диаграмма 1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Столбчатая диаграмма 2</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QRadioButton" name="radioProgressMonth">
              <property name="text">
               <string>Прогресс в течение месяца</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButton" name="radioSumMonth">
              <property name="text">
               <string>Сумма за текущий месяц всех привычек</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="radioProgressYear">
              <property name="text">
               <string>Прогресс в течение года</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QRadioButton" name="radioSumYear">
              <property name="text">
               <string>Сумма за каждый месяц в этом году</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Выбор даты</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="calendButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/img/calendar.png</normaloff>:/icons/img/calendar.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="Date_month">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="displayFormat">
         <string>MMMM</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="Date_year">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="displayFormat">
         <string>yyyy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="findButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Открыть выбранную дату</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="prevMonthButton">
        <property name="text">
         <string>Предыдущий месяц</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nextMonthButton">
        <property name="text">
         <string>Следующий месяц</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>934</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
